doc/bucket-class-crd.md

```diff
@@ -1,73 +1,83 @@
-The operator cli currently only supports a single tier placement-policy for a bucket-class.
-Upon creating regular buckets, the user will first need to create a placement-bucketclass which contains placemant policy.
-Upon creating namespace buckets, the user will first need to create a namespace-bucketclass which contains namespace policy.
-A namespace bucket class of type cache must contain both Placement-policy or Namespace-policy.
-A namespace bucket class of type single/multi must contain Namespace-policy.
-YAML must be used to create a bucket-class with a placement-policy that has multiple tiers.
-Placement-policy is case sensitive and should be of value (Mirror|Spread).
-Namespace-policy is case sensitive.
-
-For more information on using bucket-classes from S3 see [S3 Account](s3-account.md).
-
-# Definitions
-
-- CRD: [noobaa.io_bucketclasses_crd.yaml](../deploy/crds/noobaa.io_bucketclasses_crd.yaml)
-- CR: [noobaa.io_v1alpha1_bucketclass_cr.yaml](../deploy/crds/noobaa.io_v1alpha1_bucketclass_cr.yaml)
+```
+replicationPolicy: 
+  [{ 
+    "rule_id": "rule-1",
+    "destination_bucket": "first.bucket",
+    "filter": 
+      {"prefix": "ab."}
+  }]
+```
 
+## Constraints:
+- A backing store name may appear in more than one bucket class but may not appear more than once in a certain bucket class.
+- The operator CLI currently only supports a single tier placement policy for a bucket class.
+- Thus, YAML must be used to create a bucket class with a placement policy that has multiple tiers.
+- Upon creating standard buckets, the user will first need to create a placement bucketclass which contains a placemant policy.
+- Upon creating namespace buckets, the user will first need to create a namespace bucketclass which contains a namespace policy.
+- A namespace bucket class of type cache must contain both a placement and a namespace policy.
+- A namespace bucket class of type single/multi must contain a namespace policy.
+- Placement policy is case sensitive and should be of value `Mirror` or `Spread` when more than one backingstore is provided.
+- Namespace policy is case sensitive and should be of values `Single`, `Multi` or `Cache`.
 
-# Reconcile
 
-- The operator will verify that bucket-class is valid - i.e. that the backing-stores exist and can be used.
-- Changes to a bucket-class spec will be propagated to buckets that were instantiated from it.
-- Other than that the bucket-class is passive, just waiting there for new buckets to use it.
+# Reconciliation
+- The operator will verify that the bucket class is valid - i.e. that the backingstores and namespacestores exist and can be accessed and used.
+- Changes to a bucket class spec will be propagated to buckets that were instantiated from it.
+- Other than that the bucket class is passive, just waiting there for new buckets to use it.
 
-# Read Status
+# Resource Status
+It is possible to check a resource's status in several ways, including:
+- `kubectl get bucketclass <NAME> -o yaml`
```

Comment:
Maybe worth mentioning that NooBaa (now) supports BucketClass in any namespace so in order to find all of the BucketClass, use `-A`.